src/Kernel.php

```diff
@@ -33,6 +33,6 @@ protected function getContainerBaseClass(): string
 
     private function isTestEnvironment(): bool
     {
-        return 0 === strpos($this->getEnvironment(), 'test');
+        return str_contains($this->getEnvironment(), 'test');
```

Comment:
We should go with `str_start_with` instead of looking for `test` in the whole env name.